phpBB/styles/prosilver/theme/forms.css

```diff
@@ -326,7 +332,7 @@ textarea.inputbox {
 }
 
 input[type="number"] {
-	-moz-padding-end: 0;
+	-moz-padding-end: inherit;
 }
 
```

Comment:
this was a change that was merged that you are undoing please revert it to the current head